IonDotnet/Builders/IonBinaryWriterBuilder.cs

```diff
@@ -12,11 +12,18 @@ public static class IonBinaryWriterBuilder
         /// </summary>
         /// <param name="outputStream">Output stream.</param>
         /// <param name="imports">Imported symbol tables used to encode symbols.</param>
+        /// <param name="isFloatBinary64Enabled">Write float value as 64 bit.</param>
         /// <returns>A new Ion writer.</returns>
-        public static IIonWriter Build(Stream outputStream, IEnumerable<ISymbolTable> imports = null)
+        public static IIonWriter Build(
+            Stream outputStream,
+            IEnumerable<ISymbolTable> imports = null,
+            bool isFloatBinary64Enabled = false)
```

Comment:
Let's call this `forceFloat64`. 'Binary' is redundant since this is already a property of `IonBinaryWriterBuilder`. And even when this property is false, float 64 is still 'enabled'; 64-bit floats will be used when the value doesn't fit in a 32-bit float with full fidelity.

I also recommend changing the documentation to "Always write float values in 64 bits. When false, float values will be written in 32 bits when it is possible to do so without losing fidelity."

Reply:
Addressed